core/src/main/java/org/n52/sta/serdes/json/AbstractJSONObservation.java

```diff
@@ -170,6 +171,8 @@ protected T createPostEntity() {
         self.setIdentifier(identifier);
         self.setStaIdentifier(identifier);
 
+        self.setId(new Random().nextLong());
```

Comment:
This is just a workaround for the Entity using a wrong `equals()` method and therefore collapsing all Entities into one once we use a HashSet for storage. We had this problem before, but unfortunately we cannot force subclasses to overwrite `equals()` and therefore it was forgotten and this error was reintroduced.

Root cause is fixed in:
https://github.com/52North/sensorweb-server-db-model/commit/5deb40cbae6bb0d3f9ff6a56a58bf8b68ae1162c